frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,80 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+
+
+        @regression @to-do
+        Scenario: Setting Developer preference for perspective: UP-01-TC01
+            Given user is at admin perspective
+             When user clicks on user dropdown on masthead and selects "User Preference"
+              And user changes user preference "Perspective" dropdown to "Developer"
+              And user reloads the console without perspective.
+             Then user sees the "Developer" perspective
+
+
+        @regression @to-do
+        Scenario: Setting a preference for a project: UP-01-TC02
+            Given user has created project "test-preference1"
+             When usewr selects "All projets" from the project menu
+              And user clicks on user dropdown on masthead and selects "User Preference"
+              And user clicks on Project dropdown on User Preferences page
+              And user searches and selects project "test-preference1" from the dropdown
+              And user reloads the console
+             Then user can see "test-preference1" option selected in project namespace
+
+
+        @regression @to-do
+        Scenario: Creating project with project preference: UP-01-TC03
+             When user clicks on user dropdown on masthead and selects "User Preference"
+              And user clicks on Project dropdown on User Preferences page
+              And user types project "test-preference2" in search bar
+              And user clicks on the Create project option from the dropdown
+              And user can see "test-preference2" option selected in project namespace
+              And user reloads the console
+              And user can see "test-preference2" option selected in project namespace
```

Comment:
I expect you mean "...in project dropdown" instead of "...in project namespace". But you can make it more clear with a sentence like this. Its up to you, but at least change to project dropdown or something similar.

```suggestion
              And user can see that project "test-preference2" is selected
```

Reply:
Updated on similar ines